blocks/identity-block/intl.json

```diff
@@ -276,7 +276,7 @@
 	},
 	"identity-block.login-links-signup": {
 		"de": "Sign up for an account.",
-		"en": "Sign up for an account.",
+		"en": "Don't have an account? Sign up",
```

Comment:
Can we update the others as well so we get proper translations whenever that's issued?

Reply:
I tried, but for some reason when I download the files from localize it keeps the old values. I don't know if I'm missing a setting when importing/exporting. I had the `Replace with base language` option checked, but it seems to not see the update.

Reply:
interesting... if you are going to add in the error test, I'll check this when you are done with that,  Overall it looks ok to me atm.

Reply:
The error test is needed for the log in and sign up forms, this one just outputs links so there should be no need for that.

Reply:
@nschubach , see my note above. Is there anything else you think needs to be changed, or can this be approved?